### PR TITLE
README: Clarified local testing process

### PR DESCRIPTION
**Pull request by JamesMGreene**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,19 +112,25 @@ $ ./packages/percolatestudio-migrations/migrate.sh latest --settings ./setting.j
...
   ```
 
 ## Contributing
 
-Write some code. Write some tests. To run the tests, do:
+1. Write some code.
+2. Write some tests.
+3. From this package's local directory, start the test runner:
+
+    ```
```
Comment (JamesMGreene) [inline README.md]: FWIW, I chose not to use the `sh` language indicator on this fenced code block as it makes "test" get inappropriate syntax highlighting.

File: README.md [hunk 2]
```diff
@@ -112,19 +112,25 @@ $ ./packages/percolatestudio-migrations/migrate.sh latest --settings ./setting.j
 
   Migrations set a lock when they are migrating, to prevent multiple instances to prevent multiple instances of your clustered app from running migrations simultaneously. If you migrations throw an exception, you will need to manually remove the lock (and ensure your db is still consistent) before re-running the migration. Update the migrations collection like this:
 
-  ``` sh
-  meteor mongo
+  ```
```
Comment (JamesMGreene) [inline README.md]: I chose not to remove the sh language indicator from this fenced code block as only the first line is actually a true shell command.

File: README.md [hunk 3]
```diff
@@ -112,19 +112,25 @@ $ ./packages/percolatestudio-migrations/migrate.sh latest --settings ./setting.j
...
+  $ meteor mongo
```
Comment (JamesMGreene) [inline README.md]: Hopefully the addition of this `$ ` prefix and the extra line break afterward will help clarify for any newbies that the first line is a terminal/Bash shell command whereas the following lines take place in the Mongo shell that opens up thereafter.

**Comment by zol** [comment]
Thanks @JamesMGreene

**Pull request merged**